backend/geonature/core/auth/routes.py

```diff
@@ -112,6 +112,15 @@ def loginCas():
                 "id_organisme": organism_id,
             }
             response.set_cookie("current_user", str(current_user), expires=cookie_exp)
+
+            # Log the user in
+            user = db.session.execute(
+                sa.select(models.User)
+                .where(models.User.identifiant == current_user["user_login"])
+                .where(models.User.filter_by_app())
+            ).scalar_one()
```

Comment:
**Suggestion:** Consider using SQLAlchemy's ORM querying style for better readability and maintainability. [maintainability]
```suggestion
            user = models.User.query.filter_by(identifiant=current_user["user_login"]).filter_by_app().one()
```

Reply:
SQLA 1.3 donc non :P